### ISSUE TITLE
Doesn't work on Wayland natively

### ISSUE THREAD

**Issue by mxkrsv** [flathub/sh.ppy.osulazer#4]
If envvar `SDL_VIDEODRIVER` is `wayland`, dotnet throws that error:
```
Unhandled exception. System.InvalidOperationException: Sequence contains no elements
   at System.Linq.ThrowHelper.ThrowNoElementsException()
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at osu.Framework.Platform.SDL2DesktopWindow.get_PrimaryDisplay()
   at osu.Framework.Platform.SDL2DesktopWindow.<SetupWindow>b__165_1(ValueChangedEvent`1 evt)
   at osu.Framework.Bindables.Bindable`1.BindValueChanged(Action`1 onChange, Boolean runOnceImmediately)
   at osu.Framework.Platform.SDL2DesktopWindow.SetupWindow(FrameworkConfigManager config)
   at osu.Framework.Platform.GameHost.Run(Game game)
   at osu.Desktop.Program.Main(String[] args) in /run/build/osu-lazer/osu.Desktop/Program.cs:line 86
```

**Comment by Alexmitter** [comment]
Can not reproduce, I also start this flatpak with SDL_VIDEODRIVER=wayland set via Flatseal. And it starts just fine, just to be sure I just checked again if it really runs in wayland mode and it does.

Did you give the flatpak --socket=wayland? Because by default it only set --socket=x11 and if you did not also set that it will crash, also --device=dri may be needed, both can be easily set with flatseal.

**Comment by BiteDasher** [comment]
Resolved in 4d8a1de335be3e144c8e3f9ffb05b942181888ae
